### ISSUE TITLE
Slider without popup

### ISSUE THREAD

**Issue by semau** [dimsemenov/PhotoSwipe#844]
Hi,
First, thanks for this great slider.
It's possible to disable the popup? i.e, the gallery will be on my site and not need to open a popup to magnify or move between photos - all this things will be in my site directly.
Thanks,
Miki
p.s: sorry about my lame English, working on it :)

**Comment by dimsemenov** [comment]
Stretch `.pswp` element via CSS to size that you need and just initialize PhotoSwipe as usual with options:

```js
closeOnScroll:false, 
pinchToClose:false, 
closeOnVerticalDrag:false, 

closeEl:false, 
tapToClose:false,
clickToCloseNonZoomable:false,
closeElClasses:[]
```

- Also hide close button via CSS.
- And note that PhotoSwipe blocks vertical scroll on mobile, so if you have some other content above/below PhotoSwipe – look for another gallery script (e.g. [this one](http://dimsemenov.com/plugins/royal-slider/gallery/)).

**Comment by semau** [comment]
>And note that PhotoSwipe blocks vertical scroll on mobile, so if you have some other content above/below PhotoSwipe.

But why? The zoom appears only after double click. Why are you block vertical scroll? Which functionality you add replaces the vertical scroll? 
I really like this slider.. it's exactly what I need. 
I'm also developer  so if need little change just aim me plz.
Thanks! Miki
P.s: if I will write it well I can PR it later.

**Comment by dimsemenov** [comment]
There are some issues related to preventDefault on touchstart/move/end events. Related: https://github.com/dimsemenov/PhotoSwipe/pull/794

If you need simple inline image scroller on a page, PhotoSwipe might not be the best choice, use some simple slideshow script like I mentioned in prev messasge, or create simple CSS based `overflow-x:scroll` scroller with fallback to something like iScroll.

**Comment by semau** [comment]
It's looks like you solved the vertical scroll issue in #794 I'm wrong?
>If you need simple inline image scroller on a page...

Are you know a inline slider that have zoom option optimize for mobile and you can change photos by swipe?

**Comment by dimsemenov** [comment]
You may use any simple scroller and just open PhotoSwipe when user taps on it. Or just make sure that native page zoom is not disabled.
